src/object/minishLight.c

```diff
@@ -0,0 +1,75 @@
+#include "global.h"
+#include "entity.h"
+
+extern u32 _call_via_r1(u32*);
```

Comment:
you should be able to get rid of this _call_via_r1 declaration, it's just a leftover